packages/wxr/README.md

```diff
@@ -10,4 +10,39 @@ Install the module
 npm install @wordpress/wxr --save-dev
 ```
 
+## Usage
```

Comment:
This should include a section about the "mission," i.e. creating a compatible WXR and not being responsible for data integrity checks.